.github/workflows/pull_request.yaml

```diff
@@ -77,7 +75,6 @@ jobs:
     name: Build
     needs: [run-ci, prettier]
```

Comment:
@lennessyy did you mean to include prettier? I didn't see a job with this label.

Reply:
nope - remove it 😅